### PR TITLE
adapt script to support release branches

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-827-9f4461703d3f9050653e559c6a8ed938d9c7ba1b-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-827-9f4461703d3f9050653e559c6a8ed938d9c7ba1b-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-827-9f4461703d3f9050653e559c6a8ed938d9c7ba1b-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-827-9f4461703d3f9050653e559c6a8ed938d9c7ba1b-x86_64.apk)

File: scripts/deploy.sh [hunk 1]
```diff
@@ -24,8 +24,8 @@ if [[ ${TRAVIS_SECURE_ENV_VARS} = true ]]; then
     echo -e "\e[31mDeploying app to pull request\e[0m"
     echo "${BODY}" | curl -u m-kuhn:${GITHUB_API_TOKEN} -X POST --data @- https://api.github.com/repos/opengisch/QField/issues/${TRAVIS_PULL_REQUEST}/comments
 
-  elif [[ -n ${TRAVIS_TAG} ]] && [[ ${TRAVIS_BRANCH} = stable ]]; then
-    # we are on a tag and on stable branch
+  elif [[ -n ${TRAVIS_TAG} ]] && [[ ${TRAVIS_BRANCH} =~ ^release-1_[0-9]+$  ]]; then
```
Comment (m-kuhn) [inline scripts/deploy.sh]: Only 1_?
How about
```suggestion
  elif [[ -n ${TRAVIS_TAG} ]] && [[ ${TRAVIS_BRANCH} =~ ^release-[0-9_]+$  ]]; then
```

Comment (3nids) [reply scripts/deploy.sh]: done

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-827-805a8de2d85547af707ac189d3fdb817c5616cdf-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-827-805a8de2d85547af707ac189d3fdb817c5616cdf-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-827-805a8de2d85547af707ac189d3fdb817c5616cdf-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-827-805a8de2d85547af707ac189d3fdb817c5616cdf-x86_64.apk)

**Pull request merged**